pom.xml

```diff
@@ -153,6 +153,10 @@
                         <compress>true</compress>
                         <index>true</index>
                     </archive>
+                    <source>1.6</source>
```

Comment:
I think we can switch to 1.7, it won't be possible to build and publish using 1.6 anymore

Reply:
Hello @lukaszlenart .

I think you are right - it's probably OK to switch source/target to 1.7 now (it only changed a little while ago from 1.5 to 1.6 due the Travis CI builds being blocked).   But maybe it would be cleaner to do that in a separate PR that only changes the POM ?

Let me know if you prefer source/target changed to 1.7 in this PR, or wait for a separate PR afterwards.

Reply:
Good point